src/ControllerStatusIO.h

```diff
@@ -128,7 +128,7 @@ extern BOOL Ros_Controller_IsPflActive();
 extern BOOL Ros_Controller_IsMpIncMoveErrorActive();
 extern BOOL Ros_Controller_IsAnyFaultActive();
 extern BOOL Ros_Controller_MasterTaskIsJobName(const char* const jobName);
-extern MotionNotReadyCode Ros_Controller_GetNotReadySubcode();
+extern MotionNotReadyCode Ros_Controller_GetNotReadySubcode(bool attemptingToStartMotion);
```

Comment:
The argument name doesn't match what's in the .c file

Reply:
I'll fix that. The name in the .c file is what I intended to use